Scripts/signin.js

```diff
@@ -0,0 +1,87 @@
+import * as enums from "./enums.js";
+import * as sign from "./sign.js";
+import * as regex from "./regex.js";
+
+const email = document.querySelector("#email");
+const pw = document.querySelector("#pw");
+const eyeIcon = document.querySelector(".eyeIcon");
+const form = document.querySelector("form");
+
+// Input focusout 시
+function formFocusOut(e) {
+  const targetName = e.target.name;
+  const targetValue = e.target.value;
```

Comment:
이 코드는 아래와 같이 축약할 수 있습니다.
```suggestion
  const { name, value } = e.target;
```